x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -61,6 +68,35 @@ const FilesBadge = ({
 
 FilesBadge.displayName = 'FilesBadge';
 
+const ObservablesBadge = ({
```

Comment:
Could you update `x-pack/plugins/cases/public/components/case_view/case_view_tabs.test.tsx` to reflect the changes in this file?